JavaScript/TypeScript.sublime-syntax

```diff
@@ -202,16 +232,17 @@ contexts:
     - include: ts-namespace-declaration
 
     - match: (?=const{{identifier_break}})
+      pop: true
       branch:
         - ts-const-declaration
         - ts-const-enum
       branch_point: ts-const-enum
 
     - match: (?=declare{{identifier_break}})
       branch:
-        - ts-declare-enum
+        - ts-declare
         - expression-statement
-      branch_point: ts-declare-enum
+      branch_point: ts-declare
```

Comment:
Little nitpick, but I'd suggest to move `branch_point` before `branch` in all cases as this is the most common order of keys used so far.